src/components/Navbar/index.tsx

```diff
@@ -36,75 +39,84 @@ function MobileNav({ signOut }: { signOut: () => void }) {
 					isMenuOpen ? 'block' : 'hidden'
 				} md:hidden space-y-1 pb-3 bg-black-dark absolute left-0 top-full w-full rounded-b-md`}
 			>
-				<Link to="/orgs" className="block px-3 py-2 text-base font-medium text-white bg-gray-900 rounded-md">
+				<Link to="/orgs" className="flex flex-row px-3 py-2 text-base font-medium text-white bg-gray-900 rounded-md">
+					<BuildingIcon className="mr-4" />
 					Organizations
 				</Link>
 				<Link
 					to="/profile"
-					className="block px-3 py-2 text-base font-medium text-gray-300 rounded-md hover:bg-gray-700 hover:text-white"
+					className="flex flex-row items-center px-3 py-2 text-base font-medium text-gray-300 rounded-md hover:bg-gray-700 hover:text-white"
 				>
+					<UserIcon className="mr-4" />
 					Profile
 				</Link>
 				<Link
 					to="/docs"
-					className="block px-3 py-2 text-base font-medium text-gray-300 rounded-md hover:bg-gray-700 hover:text-white"
+					className="flex flex-row px-3 py-2 text-base font-medium text-gray-300 rounded-md hover:bg-gray-700 hover:text-white"
 				>
-					Resources
+					<BookMarkedIcon className="mr-4" /> Resources
 				</Link>
 				<Link
 					to="#"
-					className="block px-3 py-2 text-base font-medium text-gray-300 rounded-md hover:bg-gray-700 hover:text-white"
+					className="flex flex-row px-3 py-2 text-base font-medium text-gray-300 rounded-md hover:bg-gray-700 hover:text-white"
+					onClick={notYetImplemented}
 				>
-					Theme
+					<MoonIcon className="mr-4" /> Theme
 				</Link>
 				<Link
 					to={undefined}
 					onClick={signOut}
-					className="block px-3 py-2 text-base font-medium text-gray-300 rounded-md hover:bg-gray-700 hover:text-white"
+					className="flex flex-row px-3 py-2 text-base font-medium text-gray-300 rounded-md hover:bg-gray-700 hover:text-white"
 				>
-					Sign Out
+					<LogOutIcon className="mr-4" /> Sign Out
 				</Link>
 			</div>
 		</div>
 	);
 }
 
-const DesktopNav = ({ signOut }: { signOut: () => void }) => {
+function DesktopNav({ signOut }: { signOut: () => void }) {
```

Comment:
Bike shed color choice: thoughts on standardizing on the stylistic choice here? `const [name] = () => {}` vs `function [name] {}`? I prefer `function` because it is less strict on the ordering, which lets you organize the code for readability a bit more flexibly. Plus, `function` is immediately recognizable when you're reading through the code, vs all the different stuff that can be stored in a `const`. Those are my preferences around it, take them for what you will!